tests/queries/0_stateless/03033_analyzer_resolve_from_parent_scope.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE vecs_Float32 (v Array(Float32)) ENGINE=Memory;
```

Comment:
Since we have this query in perf tests, maybe here we can add only simplified version

```sql
WITH (SELECT (1, 1)) AS a
SELECT sum(dist) FROM (SELECT L2Distance(a, (number, number)) AS dist FROM numbers(10));
```

Reply:
I decided that if I just reduce the table size it'd be enough.